### DISCUSSION TITLE
Nvidia-Omniverse模擬平臺

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#147] [General]
**新聞內容:**
除了打造Metaverse世界所需的技術引擎，Nvidia近年來也瞄準數位分身企業戰場，靠的就是Omniverse模擬平臺，提供企業所需的軟硬基礎設施，至今包括愛立信、BMW、西門子等都有用。甚至黃仁勳更誇口：「Omniverse將成為顛覆4千萬名創作設計者未來協作的設計引擎。」Nvidia現場還展示一段用AI模擬產生的行車路況影像，供自駕車做訓練，該影片使用Omniverse模擬平臺和Modulus框架在虛擬世界中建立高擬真訓練場景。
**新聞心得:**
我看了 新聞裡介紹它們的那部影片，所有用動畫做出來的畫面都好酷，尤其是後半部車子跟製作廚房的場景跟演講者的虛擬分身，取了各種角度的照片，最後做出超級真實的感覺!!
**新聞來源:**
https://www.ithome.com.tw/news/148177
![tu_dang_1-zhi_hao_-omniverse_-tu_pian_lai_yuan_-nvidia](https://user-images.githubusercontent.com/90883526/146665701-c0077482-da88-48e0-af7e-77d76dadcf6f.jpg)

**Comment by briankuo123** [comment]
真心覺得Nvidia在做的東西都很有趣，有機會我也想去那裡上班。

**Comment by wh20502** [comment]
現在回過頭來看nvidia，覺得某些想法非常先進，也許技術總有一天會實現，但是要先想到才有機會實現。
